ports/esp32/mbedtls/mbedtls_config.h

```diff
@@ -0,0 +1,4 @@
+// Time mbedtls_platform
+#define MBEDTLS_HAVE_ASM
+
+time_t platform_mbedtls_time(time_t *timer);
```

Comment:
I don't think this file will be picked up by the build, the IDF has its own config file.  So just remove it.